tvb_scripts/plot/head_plotter.py

```diff
@@ -99,7 +99,7 @@ def plot_tvb_connectivity(self, connectivity, num="weights", order_by=None, plot
             weights_figure = None
         else:
             # weights matrix
-            weights_figure = pyplot.figure(num="Connectivity weights", figsize=self.config.figures.LARGE_SIZE)
+            weights_figure = pyplot.figure(num="Connectivity weights", figsize=self.config.figures.NOTEBOOK_SIZE)
```

Comment:
This does not seem like a very good solution.
We now have in head_plotter a hard-coded size linked with a future usage in notebooks. While this would wok for the actual notebook examples, it might pose problems for future usage (e.g. when we want images saved at a very large resolution). A better solution would be to have a self.config.figures.largest_size() method that is giving us the best current choice.